src/platform/Kconfig

```diff
@@ -125,6 +125,16 @@ config IMX8ULP
 	 Select if your target platform is imx8ulp-compatible.
 	 imx.8ulp support dsp.
 
+config IMX93_A55
+	bool "Build for NXP i.MX93 arm64 architecture (Zephyr-only)"
+	select BUILD_OUTPUT_BIN
+	select ZEPHYR_LOG
+	select HOST_PTABLE
+	select DMA_DOMAIN
+	select IMX_ARM64
+	help
+	 Select if your target platform is imx93-compatible.
```

Comment:
Commit order -- this config should be introduced _after_ the code it depends on.

Reply:
Shouldn't it be the other way around though? Seems kinda odd to use a config which has yet to be introduced. Either way is fine to me as the build will fail anyways without all the commits in this PR.

Reply:
Ah fair enough. What truly matters is that we have no commit where the build fails (eventually commits where a config can never be enabled are good). So this may be fine to dismiss.